client/telemetry/src/transport.rs

```diff
@@ -89,6 +74,7 @@ pub(crate) type WsTrans = libp2p::core::transport::Boxed<
 ///
 /// For some context, we put this object around the `wasm_ext::ExtTransport` in order to make sure
 /// that each telemetry message maps to one single call to `write` in the WASM FFI.
+// TODO: (dp) Looks like we don't need this if we remove the `wasm_ext::ExtTransport`? Rework it without.
```

Comment:
@expenses @tomaka I've looked a bit deeper at this code and I don't agree with my own comment any longer, it seems like `StreamSink` does what needs doing regardless of `wasm_ext::ExtTransport`. In other words: I'm inclined to leave this as-is (well, adjust the comment above ofc).

Reply:
Indeed, `StreamSink` is necessary anyway.